index.js

```diff
@@ -0,0 +1,27 @@
+import './style.css';
+import { savedata, retrivedata } from './modules/localStorage';
+
```

Comment:
As said above imports are only required when modules are used :)

Reply:
I used the module 'localStorage.js'. Its task requirement to create a module and importing that in 'the index.js'